### PR TITLE
Multi Target netStandard2.0 beside net451 and netstandard1.5

### PR DESCRIPTION
**Pull request by Alegrowin**
### Description

Added netstandard2.0 support to rawrabbit

### Check List

- [ ] All test passed.
- [ ] Added documentation _(if applicable)_.
- [ ] Tests added to ensure functionality.
- [ ] Pull Request to `stable` branch.

### CODE REVIEW

**Comment by Alegrowin** [comment]
@pardahlman Do you want me to provide full support for net45 net451 net452 net46 net461 net462 net47 net471 net472 netstandard1.0 netstandard1.5 and netstandard2.0 as proposed by @thyams

**Comment by tristan-hyams** [comment]
461 and 472 gets my vote. Only those two.

Thoughts: @pardahlman?

File: Debug.Props [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<Project>
+	<PropertyGroup Condition="'$(Configuration)'=='Debug'">
+		<DefineConstants>TRACE;DEBUG;LIBLOG_PORTABLE</DefineConstants>
+	</PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard1.5|AnyCPU'">
+    <DefineConstants>NETSTANDARD1_5</DefineConstants>
+  </PropertyGroup>
+	  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard1.6|AnyCPU'">
+    <DefineConstants>NETSTANDARD1_6</DefineConstants>
```
Comment (pardahlman) [inline Debug.Props:9]: Do we need to define these constants? My impression is that they are set by the compiler?

File: Debug.Props [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<Project>
```
Comment (pardahlman) [inline Debug.Props:1]: Any particular reason to call this `Debug.Props`?

**Comment by pardahlman** [comment]
The reason RawRabbit targets `net451` and `netstandard1_5` is because [the official `RabbitMQ.Client`](https://www.nuget.org/packages/RabbitMQ.Client) does so. I've been thinking about targeting `netstandard2_0`, but why target 461 and 472? 🤔

**Pull request closed without merging**